doc/apiary/v2/fiware-ngsi-simple.apib

```diff
@@ -0,0 +1,828 @@
+
++ Parameters
+    + entityId: Bcn_Welt (required, string) - Entity ID
+
++ Request (application/json)
+
+        {
+            "temperature": 21.7,
+            "relativeHumidity": {
+                "value": 60,
+                "accuracy": 1
+            },
+            "seatsNumber": 12
+        }
+
++ Response 200
+
+### Get a single attribute value [GET /v2/entities/{entityId}/attrs/{attrId}]
+
++ Parameters
+    + entityId: Bcn_Welt (required, string) - Entity ID
+    + attrId: temperature (required, string) - Attribute to be retrieved.
+
++ Response 200 (text/plain)
+
+        21.7
+
+### Get a single attribute value as JSON [GET /v2/entities/{entityId}/attrs/{attrId}]
+
++ Parameters
+    + entityId: Bcn_Welt (required, string) - Entity ID
+    + attrId: temperature (required, string) - Attribute to be retrieved.
+
++ Request
+
+    + Headers
+
+            Accept: application/json
+
++ Response 200 (application/json)
+
+        {
+            "value": 21.7
+        }
+
+### Get a single attribute value. Not exists [GET /v2/entities/{entityId}/attrs/{attrId}]
+
++ Parameters
+    + entityId: Bcn_Welt (required, string) - Entity ID
+    + attrId: mistake (required, string) - Attribute to be retrieved.
+
++ Response 404 (application/json)
+
+        {
+            "code": 4567,
+            "message": "Not found",
+            "description": "The entity does not have such an attribute"
+        }
+
+### Update an attribute value [PUT /v2/entities/{entityId}/attrs/{attrId}]
+
++ Parameters
+    + entityId: Bcn_Welt (required, string) - Entity ID
+    + attrId: temperature (required, string) - Attribute ID
+
++ Request (text/plain)
+
+        17.9
+
++ Response 200
+
+### Update an attribute value - JSON [PUT /v2/entities/{entityId}/attrs/{attrId}]
+
++ Parameters
+    + entityId: Bcn_Welt (required, string) - Entity ID
+    + attrId: temperature (required, string) - Attribute ID
+
++ Request (application/json)
+
+        {
+            "value": 25.0,
+            "unitCode": "CEL"
+        }
+
++ Response 200
+
+
+### Remove a single attribute [DELETE /v2/entities/{entityId}/attrs/{attrId}]
+
++ Parameters
+    + entityId: Bcn_Welt (required, string) - Entity ID
+    + attrId: temperature (required, string) - Attribute ID
+
++ Response 204
+
+
+## Context Entity Subscriptions [/v2/entities/{entityId}/subscriptions]
+
+### List all subscriptions present in the system [GET /v2/subscriptions]
+
++ Response 200
+
+        [
+            {
+                "id": "abcdefg",
+                "entity": {
+                    "id": "Bcn_Welt",
+                    "type": "Room"
+                },
+                "attributes": [
+                    "temperature"
+                ],
+                "callback": "http://localhost:1234",
+                "options": {
+                    "duration": "1M",
+                    "condition": "ONCHANGE",
+                    "throttling": "PT5S"
+                }
+            },
```

Comment:
oh yes, well spotted @kzangeli . Landing as per Fermin's comment. and working towards a subsequent version that fixes that issue